src/python/cli_new/lib/cli/plugins/task/main.py

```diff
@@ -106,12 +108,13 @@ def list(self, argv):
         # pylint: disable=unused-argument
         try:
             master = self.config.master()
+            config = self.config
```

Comment:
Why store it here and not directly pass `self.config` below?
```
        try:
            tasks = get_tasks(master, config)
```

Reply:
I thinks it's easier to read.

Reply:
OK!